chat/irc.properties

```diff
@@ -80,24 +78,24 @@ message.whoisEntry=\ua0\ua0\ua0\ua0%1$S: %2$S
 message.unknownNick=„%S” nie jest znanym pseudonimem użytkownika.
 message.channelKeyAdded=Użytkownik %1$S zmienił hasło dla kanału na „%2$S”.
 message.channelKeyRemoved=Użytkownik %S usunął hasło kanału.
-message.banMasks=Użytkownicy łączący się z następujących adresów są niedopuszczani do „%S”:
-message.noBanMasks=Brak wykluczonych adresów dla „%S”.
-message.banMaskAdded=Użytkownicy łączący się z adresów pasujących do „%1$S” zostali wykluczeni przez %2$S.
-message.banMaskRemoved=Użytkownicy łączący się z adresów pasujących do „%1$S” nie będą dłużej niedopuszczani przez %2$S.
+message.banMasks=Użytkownicy łączący się z następujących adresów są zbanowani z „%S”:
```

Comment:
I don't like [whole zbanowano change](https://github.com/piotrdrag/mozilla-l10n-pl/commit/706864f92fb935c7ec3a2b1dd26ac6e34295a8c2) and don't see merits for it.